### PR TITLE
Redesigned DIFM New/Existing Site Step

### PR DESCRIPTION
**Pull request by aneeshd16**
#### Changes proposed in this Pull Request

* Redesigns the New/Existing Site screen according to the new designs in pdh1Xd-FG-p2
* I have deleted the custom components and replaced them with `IntentScreen` from the `@automattic/onboarding` package. 

image

**Loading animation**

https://user-images.githubusercontent.com/5436027/157638294-84055a9d-12b9-4765-b632-a58fdd91d0b2.mp4

#### Testing instructions

* Log in as a user who has more than 1 existing site.
* Go to `/start/do-it-for-me?flags=force-sympathy`. (`flags=force-sympathy` ensures that local browser storage is cleared)
* Confirm that you can see the placeholder while the product list is loading.
* Confirm that the screen matches the design shared in the link above.
* Confirm that the price for the DIFM product is displayed correctly according to the user currency.

Related to 0-as-1201912426423457/f

### CODE REVIEW

File: client/signup/icons/index.tsx [hunk 1]
```diff
@@ -217,3 +217,38 @@ export const shoppingCart: ReactElement = (
 		/>
 	</svg>
 );
+
+export const award: ReactElement = (
```
Comment (ddc22) [inline client/signup/icons/index.tsx:221]: Nit convention, React component should start with a capital. Ditto below.

**Review by ddc22 — APPROVED** [review]
Minor comment below but,
LGTM 🚢

File: client/signup/icons/index.tsx (hunk 1 above)
Comment (aneeshd16) [reply client/signup/icons/index.tsx:221]: Agree with you, but this file seems to be using a different convention. 🙂

Comment (ddc22) [reply client/signup/icons/index.tsx:221]: Haha missed that! :)
Still disagree with this convention but well what can we do...

**Comment by ddc22** [comment]
Works as explained, however do you think it's better to probably include a placeholder for the **entire text**  than the focussed placeholder. It might confuse the customer why the price is only hidden in the beginning.

If we do it for the entire text they have no context to infer any reason.

image

This is minor feedback, feel free to SHIP these changes.

**Comment by aneeshd16** [comment]
Thank you for the review, @jdc91! We can take up any design improvements in a follow-up PR.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7173590

Thank you @aneeshd16 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7173590).

**Pull request merged**